packages/editor/src/lib/config/TLUserPreferences.ts

```diff
@@ -13,15 +13,20 @@ const USER_DATA_KEY = 'TLDRAW_USER_DATA_v3'
 export interface TLUserPreferences {
 	id: string
 	name?: string | null
-	locale?: string | null
 	color?: string | null
+	// N.B. These are duplicated in TLdrawAppUser.
+	locale?: string | null
 	animationSpeed?: number | null
 	edgeScrollSpeed?: number | null
 	colorScheme?: 'light' | 'dark' | 'system'
 	isSnapMode?: boolean | null
 	isWrapMode?: boolean | null
 	isDynamicSizeMode?: boolean | null
 	isPasteAtCursorMode?: boolean | null
+	isToolLocked?: boolean | null
+	isGridMode?: boolean | null
+	isFocusMode?: boolean | null
+	isDebugMode?: boolean | null
```

Comment:
These aren't supposed to be synced between projects / tabs, so we shouldn't add them to user preferences.

Reply:
You don't think so? I think it's worth a discussion — or at least, maybe an education for me on what I might be missing expectation-wise for users.

In my mind, these feel like App-level type prefs (say, especially focus or debug mode) but let's talk about it more in person.